pages/common/rg.md

```diff
@@ -0,0 +1,19 @@
+# ripgrep
+
+> A fast command-line search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search for pattern including all .gitignored and hidden files:
+
+`rg -uu {{pattern}}`
+
+- Search for a pattern only in a certain filetype (e.g., html, css, etc.):
+
+`rg -t {{filetype}} {{pattern}}`
+
+- Search for a pattern in files matching a glob (e.g., `README.*`):
+
+`rg {{pattern}} -g {{glob}}`
```

Comment:
@agnivade No, I think globbing is only used for file names. I've added a example for searching in just subdirs.